src/Lean/Widget/InteractiveDiagnostic.lean

```diff
@@ -207,7 +207,9 @@ partial def msgToInteractive (msgData : MessageData) (hasWidgets : Bool) (indent
         | .widget wi alt =>
           return .tag (.widget wi (← fmtToTT alt col)) default
         | .trace cls msg collapsed children => do
-          let col := col + tt.stripTags.length - 2
```

Comment:
I believe `tt.stripTags.length` used to be always `2` until it became always `0` in https://github.com/leanprover/lean4/commit/ec59e7a2c091040d3d2d53191ea2d7f9591cbfb8#diff-c247a7de43e5ee7af1aff908c4d8e11a00d2c175a1892bf8a485c5b69450a331L162-R178